x/examples/outline-connectivity-app/shared_frontend/pages/connectivity_test/index.ts

```diff
@@ -598,22 +600,24 @@ export class ConnectivityTestPage extends LitElement {
     // TODO: move language definitions to a centralized place
     return html`<main dir="${this.locale === "fa-IR" ? "rtl" : "ltr"}">
       <header class=${this.platform?.operatingSystem === OperatingSystem.IOS ? "header--ios" : "header"}>
-        <h1 class="header-text">${msg("Outline Connectivity Test")}</h1>
+        <h1 class="header-text">${msg("Transport Tester")}</h1>
```

Comment:
The idea behind the name change was that with latest changes users can now test any of the supported transports in the SDK addition to `ss`/outline. Maybe we can drop SDK and call it `Outline Transport Tester`? 

We can even keep the previous name. I think it will better to keep the title short.

Reply:
Let's drop "Outline", since this is not an official Outline product. "Connectivity Tester" is probably more user-friendly than "Transport Tester".

Reply:
Good point. Perhaps we add a line somewhere, as a subtitle or perhaps a footer, saying _Powered by the Outline SDK_? We could link to it too.

Reply:
I looked through the [branding guideline](https://github.com/Jigsaw-Code/outline-brand) and used the `Powered By Outline` SVG logo at the bottom of the page. This way we can keep the title short and still attribute to Outline SDK. I am fine with either Transport Tester or Connectivity Tester for the title, but I think the former rhymes better :)

<img width="264" alt="Screenshot 2023-12-25 at 6 45 52 PM" src="https://github.com/Jigsaw-Code/outline-sdk/assets/117060873/203638fc-784a-413e-8d5a-6ab9f73f61e6">